hack/create-dev-cluster.sh

```diff
@@ -50,6 +50,11 @@ export WORKER_MACHINE_COUNT=${WORKER_MACHINE_COUNT:-2}
 export KUBERNETES_VERSION="${KUBERNETES_VERSION:-v1.19.7}"
 export CLUSTER_TEMPLATE="${CLUSTER_TEMPLATE:-cluster-template.yaml}"
 
+# identity secret settings.
+export AZURE_CLUSTER_IDENTITY_SECRET_NAME="cluster-identity-secret"
```

Comment:
can we remove them from the other script? we shouldn't need to set them in both

Reply:
If we do that, running make create management cluster without the script would not work

Reply:
that's true but currently running make create-cluster doesn't work without the variables set below either

Reply:
ok, moved them from the other script